drake/examples/Quadrotor/CMakeLists.txt

```diff
@@ -1,9 +1,17 @@
+include_directories(${PROJECT_SOURCE_DIR}/core)
+include_directories(${PROJECT_SOURCE_DIR}/solvers)
+include_directories(${PROJECT_SOURCE_DIR}/systems)
+include_directories(${PROJECT_SOURCE_DIR}/systems/frames)
+include_directories(${PROJECT_SOURCE_DIR}/systems/plants)
+include_directories(${PROJECT_SOURCE_DIR}/systems/plants/joints)
+include_directories(${PROJECT_SOURCE_DIR}/systems/controllers)
+include_directories(${PROJECT_SOURCE_DIR}/util)
```

Comment:
these should not be here anymore, since @tkoolen switched the way we do includes.